glyphs2fontir/src/source.rs

```diff
@@ -137,6 +144,76 @@ impl GlyphsIrSource {
     }
 }
 
+struct StaticMetadataWork {
+    font: Arc<Font>,
+}
+
+impl Work for StaticMetadataWork {
+    fn exec(&self, context: &Context) -> Result<(), WorkError> {
+        let font = self.font.as_ref();
+        debug!("Static metadata for {}", font.family_name);
+
+        let mut axis_values = Vec::new();
+        for master in font.font_master.iter() {
+            master
+                .axes_values
+                .as_ref()
+                .ok_or_else(|| WorkError::InconsistentAxisDefinitions)?
```

Comment:
what are 'inconsistent' axis definitions exactly?

Reply:
I don't think this exists anymore.